extensions/cornerstone/src/getPanelModule.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-
 import { Toolbox } from '@ohif/ui-next';
+
 import PanelSegmentation from './panels/PanelSegmentation';
 import ActiveViewportWindowLevel from './components/ActiveViewportWindowLevel';
-import PanelMeasurement from './panels/PanelMeasurement';
+import PanelMeasurement from './panels/PanelMeasurementWithNodeComponent';
```

Comment:
Can we avoid renaming this?  I agree the new name is more descriptive, but avoiding the renaming makes merges cleaner overall, and people get used to specific names.